app/src/main/res/layout/fragment_buttons_changeable.xml

```diff
@@ -11,15 +11,15 @@
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
     android:text="Short Text"
-    app:buttonIcon="@drawable/bpk_close"
+    app:buttonIcon="@drawable/bpk_close_sm"
     android:layout_marginBottom="@dimen/bpkSpacingBase" />
 
   <net.skyscanner.backpack.button.BpkButton
     android:id="@+id/button_decrease"
     android:layout_width="wrap_content"
```

Comment:
This should be match_parent for testing purposes

Reply:
this hasn't changed from before - isn't this testing that the button size decreases as the text size decreases with `wrap_content`?